mautic/mautic
# Fixed a BC change that broke batch/token/personalization support for related transports

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

A 2.12 BC/deprecation change broke support for batching contacts through Sparkpost and Mandrill. This PR fixes that along with adds a test to ensure BC is kept. 

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. Not visibly replicable without hacking code to check if metadata is populated

#### Steps to test this PR:
1. Send through sparkpost to ensure it continues to work
2. Run the new test
